packages/host/src/AppDrawer.stories.tsx

```diff
@@ -32,3 +31,8 @@ export const Portuguese = Template.bind({});
 Portuguese.args = {
   locale: 'pt',
 };
+
+export const Arabic = Template.bind({});
+Arabic.args = {
+  locale: 'ar',
+};
```

Comment:
the rtl in the storybook isn't working in that the drawer isn't going to the right, but everything else is switching up